types/build.gradle

```diff
@@ -0,0 +1,19 @@
+dependencies {
+    compileOnly libs.jetbrains.annotations.get()
```

Comment:
```suggestion
    compileOnly libs.jetbrains.annotations
```

Reply:
On newer intellij versions, this seems to either work fine for `.get()` or no `.get()`, _or_ (as seems to be the case for this one) gets no IDE support -- maybe because the version catalog is in code, or maybe because of the includeBuild? It's unclear